infra/growth/migrations/20190618222143-add-growth-event-mobile-install.js

```diff
@@ -0,0 +1,14 @@
+'use strict'
+
+module.exports = {
+  up: queryInterface => {
+    return queryInterface.sequelize.query(`
+      ALTER TYPE enum_growth_event_type ADD VALUE 'MobileAppInstalled';
```

Comment:
Not sure what we call this. `MobileAppInstalled` isn't really true, its more like `AccountCreatedInMobileApp`, probably doesn't matter :)

Reply:
Good point. Let me rename this for clarity.